bitnami/prometheus-operator/crds/crd-alertmanager.yaml

```diff
@@ -679,17 +695,19 @@ spec:
                             exists or not. Defaults to "".'
                           type: string
                         valueFrom:
-                          description: EnvVarSource represents a source for the value
-                            of an EnvVar.
+                          description: Source for the environment variable's value.
+                            Cannot be used if value is not empty.
                           properties:
                             configMapKeyRef:
-                              description: Selects a key from a ConfigMap.
+                              description: Selects a key of a ConfigMap.
                               properties:
                                 key:
                                   description: The key to select.
                                   type: string
                                 name:
-                                  description: 'Name of the referent. More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names'
+                                  description: 'Name of the referent. More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names
+                                    TODO: Add other useful fields. apiVersion, kind,
```

Comment:
I would not leave this comment here

Reply:
@miguelaeh I used the following as a source.
[https://github.com/coreos/prometheus-operator/tree/release-0.38/example/prometheus-operator-crd](https://github.com/coreos/prometheus-operator/tree/release-0.38/example/prometheus-operator-crd)

Reply:
Used this regex `\s*TODO:[\w\s\.,\?]*'\n` --> `'\n` to replace every to do with nothing.